graphql_compiler/compiler/compiler_frontend.py

```diff
@@ -525,6 +526,19 @@ def _compile_vertex_ast(schema, current_schema_type, ast,
     return basic_blocks
 
 
+def _validate_fold_has_outputs(fold_data, outputs):
+    # At least one output in the outputs list must point to the fold_data,
+    # or the scope corresponding to fold_data had no @outputs and is illegal.
+    for output in outputs.values():
+        if output['fold'] is fold_data:
+            return True
+    
+    raise GraphQLCompilationError(u'Each @fold scope must contain at least one field '
+                                  u'marked @output. Encountered a @fold with no outputs '
+                                  u'at query location: {}'.format(fold_data['root']))
+
+
+
```

Comment:
Hmm it seems that I forgot to add `flake8` to the project -- my bad! Anyway, the 3 consecutive blank lines don't pass flake8:
```
$ flake8 ./graphql_compiler/
./graphql_compiler/compiler/compiler_frontend.py:541:1: E303 too many blank lines (3)
```